test/controllers/event_controller_test.exs

```diff
@@ -17,4 +17,21 @@ defmodule Dwylbot.EventTestController do
     conn = post conn, "/event/new", payload
     assert json_response(conn, 200)
   end
+
+  test "POST /event/new - new issue without description", %{conn: conn} do
+    payload = "./test/fixtures/no_description.json"
+    |> File.read!()
+    |> PP.parse!()
+    conn = post conn, "/event/new", payload
+    assert json_response(conn, 200)
+  end
+
+  test "POST /event/new - unassigned with inprogress", %{conn: conn} do
+    payload = "./test/fixtures/unassigned_inprogress.json"
+    |> File.read!()
+    |> PP.parse!()
+    conn = post conn, "/event/new", payload
+    assert json_response(conn, 200)
+  end
```

Comment:
At this point I would abstract out these 4 tests into 1 test:

```elixir
@fixtures ~w(add_label inprogress no_description unassigned_inprogress) |> Enum.map(&("./test/fixtures#{&1}.json"
test "POST /event/new", %{conn: conn} do
  for fixture <- @fixtures do
    payload = fixture |> File.read! |> PP.parse
    conn = post conn, "/event/new", payload
    assert json_response(conn, 200)
  end
end
```

Or something like that